.github/CONTRIBUTING.md

```diff
@@ -192,6 +193,14 @@ Note: VSC requires an [extension](https://marketplace.visualstudio.com/items?ite
 
 Github actions that require additional configuration are disabled on the repository until ACTION_ENABLER secret is created with non-empty value.
 
+## Can I add a Git submodule?
+### No.
+Submodules are buggy, error prone, and every time we've tried to use them, things break and it has to be removed.
+See:
+https://github.com/tgstation/tgstation/pull/28871
+https://github.com/tgstation/tgstation/pull/28902
+https://github.com/tgstation/tgstation/pull/29091
```

Comment:
not the right place for this, also weird thing to put in the guidelines if none's adding submodules to this repo